source/quickstart.txt

```diff
@@ -73,6 +73,12 @@ Follow the instructions below to setup {+c2c-product-name+}.
       <create-new-cluster>` or to create a :ref:`self-managed cluster
       <server-replica-set-deploy>`.
 
+      .. note:: 
+
+         To minimize latency after sync completes, set 
```

Comment:
**Suggestion**

Turn the body of this note into an include for additional usage in the docs.